your_package/Pose2D target
string ref_frame
bool wait_for_success
---
bool success